roles/engineer.md

```diff
@@ -18,6 +16,21 @@ We expect someone in the role:
 - to be autonomously delivering features of a small nature within 3 months
 - to be autonomously delivering features of a medium nature within 6 months
 
+
+## Academy Engineer role
+
+We need someone:
+
+- to train to be ready to join a delivery team following graduation
+
+We expect someone in the role:
+
+- to be autonomously delivering features of a micro nature before graduation
```

Comment:
"low complexity" => implications on size of domain, customer's understanding of their own domain / needs / desires? Solution for the problem is of low technical complexity - requires very few moving parts i.e. data is already in near-enough correct format, need presentational work?